packages/react-codemirror/package.json

```diff
@@ -28,7 +28,7 @@
     "test": "vitest run",
     "test:e2e": "playwright test -c playwright-ct.config.ts",
     "test:e2e-ui": "playwright test -c playwright-ct.config.ts --ui",
-    "benchmark": "BENCHMARKING=true npx playwright test -c playwright-ct.config.ts -g \"benchmarking & performance test session\""
```

Comment:
Don't we need `pnpx` here?

Reply:
`pnpx` will always [download the latest from the internet](https://pnpm.io/cli/dlx) and not use the locally installed version

Reply:
For calling dependencies, we don't need a prefix (see how we call eslint, esbuild, playwright in the above command etc)